### DISCUSSION TITLE
使用 Beancount 管理家庭财务 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#143] [Announcements]
# 使用 Beancount 管理家庭财务 · BMPI

本文介绍了我用 Beancount 替换近 9 年多的挖财记账 App。

https://www.bmpi.dev/self/beancount-my-accounting-tool-v2/

**Comment by rexjiahao** [comment]
请问应该如何更新投资组合的price呢？
